### ISSUE TITLE
Can't build based on current repositories.

### ISSUE THREAD

**Issue by Clarkinator** [DieReicheErethons/Brewery#119]
I keep getting missing artifact com.griefcraft.lwc:LWCPlugin:jar:1.7.2 and me.ryanhamshire:GriefPrevention:jar:14.5.1

I can fix the GriefProtection one by unlocking the repos that's commented out but the LWC one causes issues because apparently the archive isn't a readable zip file.

**Comment by Sataniel98** [comment]
The one that is commented out is mine, I created it exactly because of this issue. It seems that GriefPrevention has its own repo, though, but our version String is wrong (14.2 works). I don't know why it has been changed, it worked perfectly fine after my commits for the 1.9 update... There's still something weird with LWC.

**Comment by Clarkinator** [comment]
Yeah, a little annoying. I tried to find the versions and such to put them in manually but I can't find the versions that you guys are using anywhere. At least not that I know of.

**Comment by Sataniel98** [comment]
The problem with LWC seems to be that I switched to compiling against EntityLWC, an actively maintained fork of LWC and someone *looks at @ProgrammerDan * thought it would be a good idea to change this back for some reason. It seems we forgot to adjust the version String: EntityLWC uses a different version than LWC (currently [4.5.0-SNAPSHOT](http://repo.mcstats.org/content/repositories/public/com/griefcraft/lwc/4.5.0-SNAPSHOT/)).

**Comment by ProgrammerDan** [comment]
@Sataniel98 Hmm, my memory is getting foggy as it's been a few months.

However, I do recall rather clearly that the GriefPrevention and EntityLWC that you'd specified did not work for me (a rather symmetric problem; I fix it so it works for me, but break it for you, but to fix it for you breaks it for me ...).

I found LWC here, seemingly maintained (latest build on 2 months old): http://ci.griefcraft.com/job/LWC/ -- it ties back to the official repository. EntityLWC seemed to be a totally different thing. I might be wrong.

And I found GriefPrevention here: https://www.spigotmc.org/resources/griefprevention.1884/

@Clarkinator Note my comment on this commit: https://github.com/DieReicheErethons/Brewery/commit/bef0dd5fff191ce29a3d6b4d2b62d690141ae782

I was also unable to find _any_ official, public repository either for the versions @Sataniel98 used; so I altered them to match the pom.xml's of the versions I was able to find, as listed above.

In both cases after downloading the jar from the official source, I then installed it using `mvn install:install-file`. 

If you guys know of official repos (the LWC one that Sataniel used didn't work, for instance, and the GriefPrevention one didn't appear official -- it seems to pick and choose which version it hosts and doesn't keep any older versions, which tells me it's just some person hosting the one that they want and is not intended for public use) I'd love to know of them too.
